src/mlpack/methods/adaboost/adaboost_model.hpp

```diff
@@ -79,7 +79,7 @@ class AdaBoostModel
   //! Modify the dimensionality of the model.
   size_t& Dimensionality() { return dimensionality; }
 
-  //! Train the model.
+  //! Train the model, treat the data is all of the numeric type.
```

Comment:
It's nice to add this comment here, but it would be more correct to say `treat the data as all numeric`.